motoko/hello-angular-motoko/canister_ids.json

```diff
@@ -0,0 +1,8 @@
+{
+  "motoko": {
+    "ic":"your-unique-canister-goes-here"
+  },
+  "www": {
+    "ic": "another-unique-canister-goes-here"
+  }
+}
```

Comment:
When you remove the above (quite convoluted) way of creating canisters, this file can be deleted.

Reply:
I think this is actually a case of "that's just how dfx works" :)

If this file is not present (or a canister id is not configured for the selected network), `dfx canister create` (or `dfx deploy`, which is basically `dfx canister create && dfx build && dfx canister install`) writes those IDs for you.

This also requires that you have a wallet configured, which is why I'd like to the quickstart guide which describes how to set up a wallet.